docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/terraform-setup.md

```diff
@@ -0,0 +1,173 @@
+---
+id: eks-terraform
+title: "EKS Terraform Setup"
+description: "Preconfigured Terraform module for quick C8 setup."
+---
+
+Terraform is a popular Infrastructure as Code (IaC) tooling that allows to manage your infrastructure with reproduceable configuration files rather than manually setting things up. We'll take advantage of this fact and provide a basic infrastructure setup for Camunda to quick-start your journey on AWS EKS.
+
+:::tip
+
+If you are complelty new to Terraform and the idea of IaC, it makes sense to have a read through the [Terraform IaC documentation](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/infrastructure-as-code) and give their [interactive quick start](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/infrastructure-as-code#quick-start) a try to get a basic understanding.
+
+:::
+
+## Prerequisites
+
+- an [AWS account](https://docs.aws.amazon.com/accounts/latest/reference/accounts-welcome.html) is required to create any resources within AWS.
+- [Terraform](https://developer.hashicorp.com/terraform/downloads) has to be installed on your system.
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the cluster.
+
+## Considerations
+
+The following does not reflect a production ready setup but is a good quick-start to get going with Camunda on AWS EKS utilizing [IaC tooling](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/infrastructure-as-code).
+
+For the simplicity of this tutorial we'll be using non best practices but refer to further documents to allow you to delve deeper into the topic.
+
+## Outcome
+
+Following this tutorial and steps, will result in
+
+- an AWS EKS Kubernetes Cluster with 4 nodes and the possibility to scale up further.
+- the [EBS CSI driver](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) is installed and configured.
```

Comment:
A user might wonder what this is used for? For the next item about managed PostgreSQL you give a reason ("used by the Camunda applications").